WordPress/src/main/java/org/wordpress/android/viewmodel/history/HistoryViewModel.kt

```diff
@@ -223,6 +223,18 @@ class HistoryViewModel @Inject constructor(
         fetchRevisions()
     }
 
+    private fun saveRevisionsToLocalDB(post: PostModel, revisions: List<RevisionModel>) {
+        revisions.forEach {
+            postStore.setLocalRevision(it, site, post)
+        }
+    }
+
+    private fun removeRevisionsFromLocalDB(post: PostModel, revisions: List<RevisionModel>) {
+        revisions.forEach {
+            postStore.deleteLocalRevision(it, site, post)
```

Comment:
If we only remove the revisions received from the endpoint, there is a chance we might have some revisions not removed, if for some reason they are not in the response. What do you think about removing all the revision for specific post?